addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderServerBridgeHandler.java

```diff
@@ -901,58 +1305,91 @@ public void updateChannel(ChannelUID channel) {
             }
 
             if (state != null) {
-                logger.debug("{}: BridgeHandler.updateChannel(): Updating channel '{}' to state='{}'",
-                        getLogIdentifier(), channel.getId(), state.toString());
                 updateState(channel.getId(), state);
             }
         } catch (Exception ex) {
```

Comment:
Please catch specific exceptions instead of catching all exceptions. That way it is more clear what exceptions need to be handled and how this is done.